components/com_contact/tmpl/contact/default_articles.php

```diff
@@ -11,18 +11,51 @@
 defined('_JEXEC') or die;
 
 use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Layout\LayoutHelper;
 use Joomla\CMS\Router\Route;
 use Joomla\Component\Content\Site\Helper\RouteHelper;
 
 ?>
 <?php if ($this->params->get('show_articles')) : ?>
 <div class="com-contact__articles contact-articles">
-    <ul class="list-unstyled">
-        <?php foreach ($this->item->articles as $article) : ?>
-            <li>
-                <?php echo HTMLHelper::_('link', Route::_(RouteHelper::getArticleRoute($article->slug, $article->catid, $article->language)), htmlspecialchars($article->title, ENT_COMPAT, 'UTF-8')); ?>
-            </li>
+    <?php if ($this->params->get('show_articles_introimage') || $this->params->get('show_articles_introtext')) :
+        foreach ($this->item->articles as $article) :
+            if ($this->params->get('show_articles_introimage')) :
+                $images     = json_decode($article->images);
+                $layoutAttr = [
+                    'src' => $images->image_intro,
+                    'alt' => empty($images->image_intro_alt) && empty($images->image_intro_alt_empty) ? false : $images->image_intro_alt,
+                ];
+            endif;
+        ?>
```

Comment:
```suggestion
            ?>
```
Don't ask me why, but the code style checker complains about this and suggests the change.

But maybe it would be even better to use not such a bit PHP block but smaller one, e.g. one for each line.

Reply:
I changed the code